app/index.ts

```diff
@@ -55,25 +55,98 @@ function createWindow() {
 //
 // Create the window when the application is ready.
 //
-app.on("ready", createWindow);
+app.on("ready", () => {
+    createWindow();
+});
 
 
 //
 // Terminates the application when all windows have been closed.
 //
 app.on("window-all-closed", () => {
-    if (process.platform !== "darwin") {
-        app.quit();
-    }
+    //if (process.platform !== "darwin") {
+    //    app.quit();
+    //}
+
+
+    // It's becoming more common in macOS to just quit the application and 
+    // Save state between launches.
+    app.quit();
 });
 
 
 //
 // Recreates the window if it has been lost while the application was inactive.
 // This will likely occur on mobile devices with limited resources.
 //
-app.on("activate", () => {
-    if (win === null) {
-        createWindow();
+// app.on("activate", () => {
+//     if (win === null) {
+//         createWindow();
+//     }
+// });
```

Comment:
Same with this. No reason to leave around commented out code. Make note of the SHA of the commit that has it elsewhere and delete.